Framework/PDFExporter/PDFExporter/Categories/UIKit/UITableView+PDFExporterDrawing.m

```diff
@@ -0,0 +1,125 @@
+//
+//  UITableView+Extension.m
+//  PDFExporter
+//
+//  Copyright © 2016 3Pillar Global. All rights reserved.
+//
+
+#import "UITableView+PDFExporterDrawing.h"
+#import "UIView+PDFExporterPageInformation.h"
+#import "CGGeometry+Additions.h"
+
+@implementation UITableView (PDFExporterExtension)
+
+- (void)drawViewWithinPageRect:(CGRect)rect {
+    CGRect drawingFrame = self.drawingFrame;
+    UIBezierPath *path = [UIBezierPath bezierPathWithRoundedRect:drawingFrame cornerRadius:self.layer.cornerRadius];
+    
+    [self drawBackgroundWithPath:path];
+    [self drawContentWithPath:path];
+    [self drawSubviewsWithPath:path withinPageRect:rect];
+    if (self.layer.borderWidth != 0) {
+        [self drawBorderWithPath:path];
+    }
+}
+
+- (void)drawSubviewsWithPath:(UIBezierPath *)path withinPageRect:(CGRect)rect {
+    CGPoint offset = CGPointZero;
+    for (CGFloat yCoordinate = CGRectGetMinY(rect);
+         yCoordinate < CGRectGetMaxY(rect);
+         yCoordinate += CGRectGetHeight(self.bounds)) {
```

Comment:
No need to call CGRectGetMaxY and CGRectGetHeight for each iteration

Reply:
What is happening here is that we're looping from minimum Y to maximum Y with using the height of bounds as step.